### ISSUE TITLE
should not escape backslash

### ISSUE THREAD

**Issue by kenan2002** [i18next/i18next-scanner#39]
#### Version
- i18next: <i>4.2.0</i>
- i18next-scanner: <i>1.5.1</i>

#### Configuration
```js
{ // i18next-scanner options
        lngs: ['zh-CN'],
        func: {
          list: ['i18next.t', 'i18n.t', 't'],
          extensions: ['.js', '.jsx']
        },
        resource: {
          savePath: `{{lng}}/${moduleName}.json`
        },
        keySeparator: false,
        nsSeparator: false
}
```

I'm using [Key Based Fallback](http://i18next.com/translate/keyBasedFallback/). If I write code like `i18next.t('first line\nsecond line')`, i18next-scanner will generate json like this:
```json
{
   "first line\\nsecond line": ""
}
```
The double backslash which escapes the original new-line character will cause the key in json file different with the original key, leading to invalid translation.

**Comment by cheton** [comment]
I will try to add support for the following escape sequences, except the octal escape.

https://mathiasbynens.be/notes/javascript-escapes
#### Single character escape sequences
* `\b` backspace
* `\f` form feed
* `\n` line feed
* `\r` carriage return
* `\t` horizontal tab
* `\v` vertical tab
* `\0` null character
* `\'` single quote
* `\"` double quote
* `\\` backslash

#### Octal escape sequences (deprecated)
`\\(?:[1-7][0-7]{0,2}|[0-7]{2,3})`

#### Hexadecimal escape sequences
`\\x[a-fA-F0-9]{2}`

#### Unicode escape sequences
`\\u[a-fA-F0-9]{4}`

**Comment by cheton** [comment]
Fixed in 1.6.0

**Comment by kenan2002** [comment]
@cheton Thank you very much for the quick response!
